wallet/walletrpc.c

```diff
@@ -407,6 +429,11 @@ static void json_listfunds(struct command *cmd, const char *buffer UNUSED,
 				     c->funding_satoshi);
 			json_add_txid(response, "funding_txid",
 				      &c->funding_txid);
+			derive_basepoints(&c->seed, &funding_pubkey, NULL, NULL, NULL);
+			pubkey_to_hash160(&funding_pubkey, &h160);
+			out = p2sh_to_base58(cmd,
+				get_chainparams(cmd->ld)->testnet, &h160);
+			json_add_string(response, "funding_address", out);
```

Comment:
Please remove this since it may lead people into thinking that it's one of their addresses. Sending to this address will lock up funds.